api/graphite.go

```diff
@@ -26,6 +26,7 @@ import (
 	tags "github.com/opentracing/opentracing-go/ext"
 	"github.com/raintank/dur"
 	"github.com/raintank/worldping-api/pkg/log"
+	"github.com/tinylib/msgp/msgp"
```

Comment:
a bit weird that this library now needs to be aware of the msgp serialization whereas in other codepaths this is handled by the callers, can we clean this up a bit?

Reply:
how about i move clusterQuery to /api/cluster.go? that one already has msgp imported anyway

Reply:
like https://github.com/grafana/metrictank/commit/6924ec6f5305aa07839303a30080816e4f1a148b